shell/platform/common/cpp/client_wrapper/standard_codec.cc

```diff
@@ -176,8 +181,10 @@ void StandardCodecSerializer::WriteValue(const EncodableValue& value,
       const auto& string_value = value.StringValue();
       size_t size = string_value.size();
       WriteSize(size, stream);
```

Comment:
Should this be included in the if statement below?

Reply:
No we always want to write the size because the format specifies type|size|contents and the reading code needs to read that zero to be able to decode an empty string.

Reply:
makes sense, thanks!